scripts/migrations/002-remove-webview-flags.test.js

```diff
@@ -3,10 +3,7 @@
  * http://creativecommons.org/publicdomain/zero/1.0/ */
```

Comment:
Are the migrations/ things still needed? @queengooborg

Reply:
We aren't really using the migration scripts for anything, but I'll defer to @ddbeck on whether we should remove them.